.github/workflows/e2e.istio.yaml

```diff
@@ -73,7 +77,7 @@ jobs:
                --set elasticsearch.replicas=1 \
                --set elasticsearch.minimumMasterNodes=1 \
                --set elasticsearch.imageTag=7.5.1 \
-               --set oap.env.SW_ENVOY_METRIC_ALS_HTTP_ANALYSIS=k8s-mesh \
+               --set oap.env.SW_ENVOY_METRIC_ALS_HTTP_ANALYSIS=${{ matrix.analyzer }} \
```

Comment:
I only note this difference between the two analyzers. But is the meta exchange active in default in Istio? I remember it isn't. At the same time, I can't see when do you activate it.

Reply:
Because `metadataExchange` is enabled when `telemetry.v2` is enabled, and `telemetry.v2` is enable by default in the `demo` profile, we install Istio with `profile=demo`, hence `metadataExchange` is enabled too